modin/logging/class_logger.py

```diff
@@ -31,10 +31,12 @@ class ClassLogger:
     This mixin must go first in class bases declaration to have the desired effect.
     """
 
+    _modin_logging_layer = "PANDAS-API"
+
     @classmethod
     def __init_subclass__(
         cls,
-        modin_layer: str = "PANDAS-API",
+        modin_layer: Optional[str] = None,
```

Comment:
Why do we need the change here on Line 39 and Line 58? Maybe I am missing something, but does just adding  `cls._modin_logging_layer = modin_layer` after defining `_modin_logging_layer` fix this issue?

If not, can we also update the function comment to reflect the `Optional[str] = None`?

Reply:
@naren-ponder I think the Pythonic way here is to default to `None`, I don't think what you describe will work because `cls` is not in scope in the signature.

Reply:
@devin-petersohn Makes sense.